client.proto

```diff
@@ -194,11 +194,46 @@ message ReplToken {
   Permissions permissions = 15;
 }
 
-// GovalTokenMetadata is information about a goval token, that can be used to
+// GovalSigningAuthority is information about a goval token, that can be used to
 // validate it. It is stored in the footer of the PASETO.
-message GovalTokenMetadata {
-  // The ID of the key that was used to sign the token.
-  string key_id = 1;
+message GovalSigningAuthority {
+  oneof cert {
+    // The ID of the public key that was used to sign the token.
+    string key_id = 1;
+
+    // The intermediate_cert that was used to sign the token.
+    string intermediate_cert = 2;
+  }
+}
+
+enum FlagClaim {
+	MINT_GOVAL_TOKEN = 0;
+	MINT_INTERMEDIATE_TOKEN = 1;
+}
+
+message CertificateClaim {
+	oneof claim {
+		string replid = 1;
+		string user = 2;
+		FlagClaim flag = 3;
+	}
+}
+
+message GovalCert {
+  // Issue timestamp. Equivalent to JWT's "iat" (Issued At) claim.  Tokens with
+  // no `iat` field will be treated as if they had been issed at the UNIX epoch
+  // (1970-01-01T00:00:00Z).
+  google.protobuf.Timestamp iat = 1;
+
+  // Expiration timestamp. Equivalent to JWT's "exp" (Expiration Time) Claim.
+  // If unset, will default to one hour after `iat`.
+  google.protobuf.Timestamp exp = 2;
+
+  // A list of claims this cert can authorize
+  repeated CertificateClaim claims = 3;
+
+  // The public key authorized to sign requests in this scope
+  string publicKey = 4;
```

Comment:
can this be a `GovalSigningAuthority`? this may be YAGNI, but since this is one of those changes that are very annoying to roll out, might as well have the capability to do this from the get-go.